api/lib/mno_enterprise/concerns/controllers/jpi/v1/teams_controller.rb

```diff
@@ -45,6 +47,10 @@ def update
     if params[:team] && params[:team][:app_instances]
       list = params[:team][:app_instances].select { |e| e != {} }
       @team.set_access_to(list)
+
+      MnoEnterprise::EventLogger.info('team_apps_update', current_user.id, 'Team apps updated', @team.organization,
+                                      {team_name: @team.name, apps: list.map{|l| l['name']}.join(',')})
```

Comment:
For the same output you could use `Array#to_sentence`
But in this case I don't mind just having:
```ruby
MnoEnterprise::EventLogger.info('team_apps_update', current_user.id, 'Team apps updated', @team, {apps: list.map {|a| a['name']}
```
The `metadata` is hash and that leave us more flexibility to format the array however we want